src/components/cosmwasm/contract_overview/index.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import classnames from 'classnames';
+import useTranslation from 'next-translate/useTranslation';
+import {
+  Typography,
+} from '@material-ui/core';
+import { useScreenSize } from '@hooks';
+import { getMiddleEllipsis } from '@utils/get_middle_ellipsis';
+import { Box } from '@components';
+import { useStyles } from './styles';
+
+const ContractOverview: React.FC<{
+  className?: string;
+  address: string;
+  deployerAddress: string;
+  label: string;
+  codeId: string;
+  block: string;
+}> = ({
+  className,
+  address,
+  deployerAddress,
+  label,
+  codeId,
+  block,
+}) => {
+  const { isDesktop } = useScreenSize();
+  const classes = useStyles();
+  const { t } = useTranslation('accounts');
+
+  return (
+    <>
+      <Box className={classnames(className, classes.root)}>
```

Comment:
I'd suggest adjusting the format of how "address" and "deployer" are displayed to be like the others: key: value, instead of  

key
value

Reply:
@mlukanova wanted them to be displayed in such format.